Mathlib/Topology/Algebra/Module/WeakDual.lean

```diff
@@ -140,19 +149,18 @@ theorem tendsto_iff_forall_eval_tendsto {l : Filter α} {f : α → WeakBilin B}
 /-- Addition in `WeakBilin B` is continuous. -/
 instance instContinuousAdd [ContinuousAdd 𝕜] : ContinuousAdd (WeakBilin B) := by
   refine' ⟨continuous_induced_rng.2 _⟩
-  refine'
-    cast (congr_arg _ _)
-      (((coeFn_continuous B).comp continuous_fst).add ((coeFn_continuous B).comp continuous_snd))
+  refine' cast (congr_arg _ _)
```

Comment:
While you're touching this line, would you like to replace `refine'` by `refine`?
There [seems to be](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/refine.20vs.20refine') consensus for this in principle (there are just too many occurrences to rewrite them all by hand).